File: front/service/src/main/java/com/coinhunt/front/data/LevelInfo.java
package com.coinhunt.front.data;

import lombok.AllArgsConstructor;
import lombok.Getter;

@AllArgsConstructor
@Getter
public class LevelInfo {

	private final Difficulty difficulty;

	private final String description;

	private final int mazeHeight;

	private final int mazeWidth;

	private final int numberOfCoins;
}

File: games/src/main/kotlin/com/coinhunt/games/persistence/domain/documents/LevelInfo.kt
package com.coinhunt.games.persistence.domain.documents

import org.bson.types.ObjectId
import org.springframework.data.annotation.Id
import org.springframework.data.mongodb.core.mapping.Document

@Document
data class LevelInfo(
    val description: String,
    val mazeHeight: Int,
    val mazeWidth: Int,
    val numberOfCoins: Int,
    @Id val id: ObjectId = ObjectId.get()
)
